open-company/open-company-web
# Add h1 and highlighter to text formatters.

Card: https://trello.com/c/s7w5BfC1

NB: interaction service will remove highlight from the body (not the text, just the background color) since it doesn't accept mark elements right now. It changes with [code-blocks PR](https://github.com/open-company/open-company-web/pull/1137)

To test:
- test H1 formatter
- test H2 formatter
- test Highlighter formatter
- use a mix of them
- make sure you can always type outside of a highlighter portion of text
